MQ2AutoSize.cpp

```diff
@@ -359,269 +536,1087 @@ PLUGIN_API void OnPulse()
+				{
+					if (AS_Config.OptAutoSave)
+					{
+						SaveINI("SizeSelf", true);
+					}
+				}
+				ImGui::EndDisabled();
+				ImGui::TableNextRow();
+				// Option: Other PCs
+				ImGui::TableNextColumn();
+				if (ImGui::Checkbox("##OptPC", &AS_Config.OptPC))
+				{
+					AS_Config.OptPC = !AS_Config.OptPC;
+					DoCommandf("/autosize pc");
+				}
+				ImGui::TableNextColumn();
+				ImGui::SetNextItemWidth(50.0f);
+				ImGui::BeginDisabled(!AS_Config.OptPC);
+				if (ImGui::SliderInt("Resize: Other player(s) (incluldes those mounted)##inputOP", &AS_Config.SizePC, 1, 30, "%d", ImGuiSliderFlags_NoInput | ImGuiSliderFlags_AlwaysClamp))
+				{
+					if (AS_Config.OptAutoSave)
+					{
+						SaveINI("SizePC", true);
+					}
+				}
+				ImGui::EndDisabled();
+				ImGui::TableNextRow();
+				// Option: Pets
+				ImGui::TableNextColumn();
+				if (ImGui::Checkbox("##OptPet", &AS_Config.OptPet))
+				{
+					AS_Config.OptPet = !AS_Config.OptPet;
+					DoCommandf("/autosize pets");
+				}
+				ImGui::TableNextColumn();
+				ImGui::PushItemWidth(50.0f);
+				ImGui::BeginDisabled(!AS_Config.OptPet);
+				if (ImGui::SliderInt("Resize: Pets##inputPS", &AS_Config.SizePet, 1, 30, "%d", ImGuiSliderFlags_NoInput | ImGuiSliderFlags_AlwaysClamp))
+				{
+					if (AS_Config.OptAutoSave)
+					{
+						SaveINI("SizePet", true);
+					}
+				}
+				ImGui::EndDisabled();
+				ImGui::TableNextRow();
+				// Option: Mercs
+				ImGui::TableNextColumn();
+				if (ImGui::Checkbox("##OptMerc", &AS_Config.OptMerc))
+				{
+					AS_Config.OptMerc = !AS_Config.OptMerc;
+					DoCommandf("/autosize mercs");
+				}
+				ImGui::TableNextColumn();
+				ImGui::PushItemWidth(50.0f);
+				ImGui::BeginDisabled(!AS_Config.OptMerc);
+				if (ImGui::SliderInt("Resize: Mercs##inputMercSize", &AS_Config.SizeMerc, 1, 30, "%d", ImGuiSliderFlags_NoInput | ImGuiSliderFlags_AlwaysClamp))
+				{
+					if (AS_Config.OptAutoSave)
+					{
+						SaveINI("SizeMerc", true);
+					}
+				}
+				ImGui::EndDisabled();
+				ImGui::TableNextRow();
+				// Option: Mounts
+				ImGui::TableNextColumn();
+				if (ImGui::Checkbox("##OptMount", &AS_Config.OptMount))
+				{
+					AS_Config.OptMount = !AS_Config.OptMount;
+					DoCommandf("/autosize mounts");
+				}
+				ImGui::TableNextColumn();
+				ImGui::PushItemWidth(50.0f);
+				ImGui::BeginDisabled(!AS_Config.OptMount);
+				if (ImGui::SliderInt("Resize: Mounts and the Player(s) on them##inputMountSize", &AS_Config.SizeMount, 1, 30, "%d", ImGuiSliderFlags_NoInput | ImGuiSliderFlags_AlwaysClamp))
+				{
+					if (AS_Config.OptAutoSave)
+					{
+						SaveINI("SizeMount", true);
+					}
+				}
+				ImGui::EndDisabled();
+				ImGui::TableNextRow();
+				// Option: Corpses
+				ImGui::TableNextColumn();
+				if (ImGui::Checkbox("##OptCorpse", &AS_Config.OptCorpse))
+				{
+					AS_Config.OptCorpse = !AS_Config.OptCorpse;
+					DoCommandf("/autosize corpse");
+				}
+				ImGui::TableNextColumn();
+				ImGui::PushItemWidth(50.0f);
```

Comment:
Seeing a ton of PushItemWidth and no PopItemWidth.

Probably better off using SetNextItemWidth instead.